garminconnect/__init__.py

```
@@ -1367,12 +1370,12 @@ def get_activity_gear(self, activity_id):
 
         return self.connectapi(url, params=params)
 
-    def get_gear_ativities(self, gearUUID):
+    def get_gear_ativities(self, gearUUID, limit = 9999):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_gear_ativities`

(ANN201)